### PR TITLE
Install Storybook

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Sets up an empty Storybook for development.

Depends on https://github.com/libero/storybook/pull/2.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,14 +17,24 @@ RUN npm install
...
+FROM storybook AS dev
+ENV NODE_ENV=development
+EXPOSE 8080
 
-CMD ["sleep", "86400"]
+CMD ["npx", "start-storybook", "--port", "8080"]
 
 HEALTHCHECK --interval=5s --timeout=1s \
-    CMD node --version
+    CMD wget --quiet --tries=1 --spider http://localhost:8080/ || exit 1
```
Comment (thewilkybarkid) [inline Dockerfile:40]: Better to use `nc` to avoid logs? (Since it's a in-browser application, shouldn't make any difference.)

Comment (giorgiosironi) [reply Dockerfile:40]: Logs shouldn't be a problem (besides the `--quiet`), though as long as the Node application is ready as soon as it starts to listen to the port doesn't make much difference (only if something is not set up correctly).

Now remembering what the `|| exit 1` is for, conforming to healthcheck standard exit codes.

File: Dockerfile [hunk 2]
```diff
@@ -17,14 +17,24 @@ RUN npm install
...
+COPY src/ src/
+
+
+
+#
+# Stage: Development environment
+#
+FROM storybook AS dev
+ENV NODE_ENV=development
+EXPOSE 8080
```
Comment (giorgiosironi) [inline Dockerfile:35]: is `80` forbidden or to be avoided? (at the host level, it usually is, in the container, may not be a problem)

File: Makefile [hunk 3]
```diff
@@ -4,6 +4,7 @@ REAL_ENV = dev
 NAME = storybook-${REAL_ENV}
 TAG = libero/storybook:${REAL_ENV}
 EXISTING_CONTAINERS = $$(docker ps --all --quiet --filter "name=${NAME}")
+MOUNT := $$([[ ${REAL_ENV} = "dev" ]] && echo "--mount type=bind,source=$$(pwd)/src/,destination=/app/src,readonly" || echo "")
```
Comment (giorgiosironi) [inline Makefile]: could use a `if`, as a failure in the `&& ...` then statement jumps to the else:
```
$ [[ "dev" == "dev" ]] && echo-typo 1 || echo 2
echo-typo: command not found
2
```

File: Dockerfile (hunk 2 above)
Comment (thewilkybarkid) [reply Dockerfile:35]: No, just trying to be consistent (since it appears in the logs at least).

File: Dockerfile (hunk 1 above)
Comment (thewilkybarkid) [reply Dockerfile:40]: (Re logs, more thinking of `prod` where it'll be Nginx, so quite visible.)

Comment (giorgiosironi) [reply Dockerfile:40]: This healthcheck seems specific to `dev`, but the Nginx `prod` configuration can turn the log off for `/ping`|`/health` or similar.

**Review by giorgiosironi — APPROVED** [review]

**Pull request merged**